web3/types.py

```diff
@@ -191,9 +191,9 @@ class LogReceipt(TypedDict):
 # syntax b/c "from" keyword not allowed w/ class construction
 TxReceipt = TypedDict("TxReceipt", {
     "blockHash": HexBytes,
-    "blockNumber": int,
+    "blockNumber": BlockNumber,
     "contractAddress": Optional[ChecksumAddress],
-    "cumulativeGasUsed": int,
+    "cumulativeGasUsed": Wei,
```

Comment:
This should still be `int`.  `gasPrice` is a `Wei` value but the actual `gas` values are all just integers.